lib/loader/loader.js

```diff
@@ -3,15 +3,37 @@ var loader, define, requireModule, require, requirejs;
 (function(global) {
   'use strict';
 
+  var heimdall = global.heimdall;
+  var {
+    define: __define,
+    require: __require,
+    exports: __exports,
+    findModule: __findModule,
+    resolve: __resolve,
+    reify,
+    findDeps,
+    modules,
+    resolveRelative,
+    pendingQueueLength
+  } = heimdall.registerMonitor('loaderjs',
+  'define',
+  'require',
+  'reify',
+  'findDeps',
+  'modules',
+  'exports',
+  'resolve',
+  'resolveRelative',
+  'findModule',
+  'pendingQueueLength');
```

Comment:
Bower looks for published packages as github tags, then github branches and then straight up repos.  There's not a lot we can do about its resolution strategy.